VAMobile/src/utils/formattingUtils.ts

```diff
@@ -68,7 +68,7 @@ export const getFormattedDateTimeYear = (dateTime: string): string => {
  */
 export const getFormattedMessageTime = (dateTime: string): string => {
   const date = DateTime.fromISO(dateTime)
-  if (DateTime.now().minus({ hours: 24 }) < date) {
+  if (DateTime.now().day === date.day && DateTime.now().month === date.month && DateTime.now().year === date.year) {
```

Comment:
Could put the DateTime.now() output into a constant to avoid calling it multiple times. Pretty minor since I doubt there are significant performance implications